src/app/dataset/depublication/depublication-row/depublication-row.component.html

```diff
@@ -0,0 +1,13 @@
+<ng-template #depublicationTemplate>
+  <ng-template #empty_date><span class="date-and-time"></span></ng-template>
+  <span class="record-url">
+    <span class="">{{ record.recordId }}</span>
+  </span>
+  <span class="status">
+    <span class="">{{ record.depublicationStatus }}</span>
```

Comment:
Should there be a translation operation here?

Reply:
That would require keys corresponding to the ENUM values 'PENDING' and 'DEPUBLISHED' (and that would have the linter complain) or some mapping logic... just to arrive at the same visual outcome.

Reply:
OK, so you're just outputting PENDING and DEPUBLISHED to the screen? Then I guess it is fine. Perhaps later at some point this should change, but we can see then.